modules/social_features/social_event/social_event.post_update.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * @file
+ * Post update functions for the Social Course Quiz module.
+ */
+
+/**
+ * Update existing events.
+ */
+function social_event_post_update_update_events(&$sandbox) {
+  /** @var \Drupal\node\NodeStorageInterface $node_storage */
+  $node_storage = \Drupal::entityTypeManager()->getStorage('node');
+
+  if (!isset($sandbox['total'])) {
+    // Get all event ids.
+    $sandbox['ids'] = $node_storage
+      ->getQuery()
```

Comment:
`$node_storage->getQuery()->accessCheck(FALSE)->condition('type', 'event')->execute();` would have been a very nice idea as this query might collection only public events.